### PR TITLE
Setup script

### PR DESCRIPTION
**Pull request by achilleas-k**
Including a setup script `setup.py`, along with `info.py` for package information.
Currently calling the package *neonix*.

Also updated the Travis configuration to use the setup script to build and test.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+try:
+    from setuptools import setup
+except:
+    from distutils.core import setup
+import re
+
+__author__ = "Achilleas Koutsou"
+
+install_requires = ["numpy>=1.3.0",
+                    "quantities>=0.9.0"]
```
Comment (stoewer) [inline setup.py]: 1. `install_requires` is not used in `setup` :-)
2. Those are already dependencies of neo, there is no need to redefine them

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      author           = AUTHOR,
+      author_email     = CONTACT,
+      url              = HOMEPAGE,
+      description      = BRIEF,
+      long_description = description_text,
+      license          = 'BSD',
+      packages         = ['neonix', 'neonix.io'],
+      tests_require    = ['nose'],
+      test_suite       = 'nose.collector',
+      requires         = ['nix', 'neo'],
```
Comment (stoewer) [inline setup.py]: Maybe it is better to use `install_requires` instead

**Comment by achilleas-k** [comment]
This is what I get from looking at and merging multiple examples instead of reading the docs.
Changes made.

**Comment by stoewer** [comment]
Is OK but can't merge

**Comment by achilleas-k** [comment]
Right. Forgot about that. One moment please.

**Comment by achilleas-k** [comment]
Good to go.

**Pull request merged**